package.json

```diff
@@ -135,6 +137,7 @@
     "i18n-js": "3.9.2",
     "ipfs-car": "^1.0.0",
     "libphonenumber-js": "^1.10.49",
+    "lodash": "^4.17.21",
```

Comment:
Why was lodash added? I feel like I missed the usage somewhere

Reply:
You're right! I'm used to always use it in my project, so maybe I had a thought of adding it but yes ending up not using it. Removed!